.rultor.yml

```diff
@@ -8,7 +12,9 @@ docker:
 merge:
   fast-forward: only
   script:
-    - python3 .ci/adjust_version_number.py bears/VERSION -b `date --utc +%Y%m%d%H%M%S`
+    - >
+      python3 .ci/adjust_version_number.py
+      bears/VERSION -b `date --utc +%Y%m%d%H%M%S`
```

Comment:
This is not the best way to split the line.
`date --utc +%Y%m%d%H%M%S` should be stored in a shell variable with a short name.

```yaml
    - DATE=`date --utc +%Y%m%d%H%M%S`
    - python3 .ci/adjust_version_number.py bears/VERSION -b "$DATE"
```